lib/movie.dart

```diff
@@ -0,0 +1,54 @@
+import 'package:flutter/material.dart';
+
+class Movie {
+  final String _title;
+  final String _originalTitle;
+  final String _overview;
+  final DateTime _releaseDate;
+  final double _voteAverage;
+  final List<String> _genres;
+  final Image _poster;
+  final Image _backdrop;
+  int _likes = 0;
+
+  String get title => _title;
+
+  Movie(this._title, this._originalTitle, this._genres, this._poster,
+      this._backdrop, this._releaseDate, this._voteAverage, this._overview);
+
+  String get originalTitle => _originalTitle;
+
+  String get overview => _overview;
+
+  DateTime get releaseDate => _releaseDate;
+
+  double get voteAverage => _voteAverage;
+
+  List<String> get genres => _genres;
+
+  Image get poster => _poster;
+
+  Image get backdrop => _backdrop;
+
+  int get likes => _likes;
```

Comment:
I think these getters are not needed. I'd make all the properties public. Just marking the properties as `final` already prevents them from being modified!

Reply:
done